docker/test_connection

```diff
@@ -44,7 +44,7 @@ for attempt in $(seq 10); do
         echo "SUCCESS"
         exit 0
     fi
-    sleep 1
+    sleep 2
```

Comment:
Looks like it has got slower for hintr to start up - not exactly sure why but this failed 3 or 4 times in a row on CI before I increased the timings